Packet++/header/TLVData.h

```diff
@@ -249,18 +260,24 @@ namespace pcpp
 		 */
 		TLVRecordType getNextTLVRecord(TLVRecordType& record, uint8_t* tlvDataBasePtr, size_t tlvDataLen)
 		{
-			if (record.isNull())
-				return TLVRecordType(NULL);
+      TLVRecordType resRec(NULL); // for NRVO optimization
```

Comment:
please use the same indentation as the rest of the file (tabs instead of spaces)

Reply:
Done